WinUIB2CApp/MSAL/MSALClientHelper.cs

```diff
@@ -0,0 +1,248 @@
+﻿using Microsoft.Identity.Client;
+using Microsoft.Identity.Client.Extensions.Msal;
+using Microsoft.IdentityModel.Abstractions;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace WinUIMSALAppB2C.MSAL
+{
+    /// <summary>
+    /// Contains methods that initialize and use the MSAL SDK
+    /// </summary>
+    /// <autogeneratedoc />
+    public class MSALClientHelper
+    {
+        public IntPtr windowHandle { get; set; }
+
+        /// <summary>
+        /// As for the Tenant, you can use a name as obtained from the azure portal, e.g. kko365.onmicrosoft.com"
+        /// </summary>
+        public readonly AzureADB2CConfig AzureADB2CConfig;
+
+        /// <summary>
+        /// Gets the authentication result (if available) from MSAL's various operations.
+        /// </summary>
+        /// <value>
+        /// The authentication result.
+        /// </value>
+        public AuthenticationResult AuthResult { get; private set; }
+
+        /// <summary>
+        /// Gets the MSAL public client application instance.
+        /// </summary>
+        /// <value>
+        /// The public client application.
+        /// </value>
+        public IPublicClientApplication PublicClientApplication { get; private set; }
+
+        private PublicClientApplicationBuilder PublicClientApplicationBuilder;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="MSALClientHelper"/> class.
+        /// </summary>
+        public MSALClientHelper(AzureADB2CConfig azureADB2CConfig)
+        {
+            this.AzureADB2CConfig = azureADB2CConfig;
```

Comment:
Please, let's follow the coding style of .NET 

https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions